### PR TITLE
Fix/08 dashboard assignment jaya

### PR DESCRIPTION
**Pull request by jbidari15**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,15 +2,21 @@ import React from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+<<<<<<< HEAD
```
Comment (trungh13) [inline src/App.js]: resolve this ?

File: src/containers/Dashboard/components/StudySync/index.js [hunk 2]
```diff
@@ -14,23 +14,21 @@ class StudySync extends Component {
   render() {
     const { dash } = this.props;
     return (
-      <div className="card col s6">
+      <div className="col s6">
         {dash.length !== undefined
           && dash.map(item => (
-            <div className="card-panel hoverable list">
+            <div className="card-panel hoverable list" key={Math.random()}>
```
Comment (trungh13) [inline src/containers/Dashboard/components/StudySync/index.js]: why Math.random ? why dont take the index ?

File: src/containers/Dashboard/index.js [hunk 3]
```diff
@@ -5,17 +5,22 @@ import Assignments from './components/Assignment';
 import DashboardComponent from '../../components/Dashboard/DashboardComponent';
 import PageTemplate from '../../components/PageTemplate';
 import './Dashboard.css';
+<<<<<<< HEAD
```
Comment (trungh13) [inline src/containers/Dashboard/index.js]: need to resolve

File: src/App.js (hunk 1 above)
Comment (jbidari15) [reply src/App.js]: resolved

**Pull request merged**